robot_descriptions/loaders/pybullet.py

```diff
@@ -46,5 +50,7 @@ def load_robot_description(
         raise ValueError(f"{description_name} is not a URDF description")
 
     pybullet.setAdditionalSearchPath(module.PACKAGE_PATH)
-    robot = pybullet.loadURDF(module.URDF_PATH)
+    robot = pybullet.loadURDF(module.URDF_PATH,
```

Comment:
Can we use the ``**kwargs`` syntax to forward all keyword arguments of `load_robot_description` to `pybullet.loadURDF`?

This way we don't need to reproduce the API here, and are more future-proof (in case the pybullet API changes at some point).

Reply:
Done :)